app/assets/v2/js/open-chat.js

```diff
@@ -4,9 +4,26 @@ const openChat = () => {
 
     $(elem).on('click', function() {
       const user = $(elem).data('openchat');
-      const url = user ? `https://chat.gitcoin.co/gitcoin/messages/@${user}` : 'https://chat.gitcoin.co/';
+      const url = user ? `https://chat.gitcoin.co/gitcoin/messages/@${user}` :
+        'https://chat.gitcoin.co/';
 
-      chatWindow = window.open(url, 'Loading', 'top=0,left=0,width=400,height=600,status=no,toolbar=no,location=no,menubar=no,titlebar=no');
+      chatWindow = window.open(
+        url, 'Loading',
+        'top=0,left=0,width=400,height=600,status=no,toolbar=no,location=no,menubar=no,titlebar=no');
+    });
+  });
+  $('[data-sponser-openchat').each(function(index, elem) {
```

Comment:
how this connects to chat is changed, there is is a new global variable available at window.chatSidebar and a helper method to open up in the new chat window. 

window.chatSidebar.chatWindow(channel)

remove the host from the url and pass just the channel like so into the helper
```window.chatSidebar.chatWindow(`gitcoin/messages/channels/company-${sponserChannel}``)```